mod-src/build.gradle.kts

```diff
@@ -20,6 +20,9 @@ plugins {
 /** The output jar files will contain this string in their names. */
 val jarName = "compiled-mod"
 
+/** Windows sucks. */
+val windows = System.getProperty("os.name").lowercase().contains("windows")
```

Comment:
Use `System.getProperty("os.name").contains("windows", true)` or `"windows" in System.getProperty("os.name").lowercase()`, please

Reply:
quiche